generate_docs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# exit on any error
+set -e
+
+# This script will
+# 1. build and install the source,
+# 2. run doxygen and create html and xml docs,
+# 3. run script to generate markdown from xml
+
+# Get the directory where of this script.
```

Comment:
Get script current directory.

("Get the directory where of this script." is confusing English)

Reply:
Oops. Thx.